superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -444,6 +444,18 @@ function ChartList(props: ChartListProps) {
       ),
       paginate: false,
     },
+    {
+      Header: t('Favorite'),
+      id: 'favorite',
+      col: 'id',
+      input: 'select',
+      operator: 'chart_is_fav',
```

Comment:
Can we use the `FilterOperators` enum here. We recently added it and haven't gotten around to converting some of the older files, but since you're in here might as well get that out of the way too